spec/routing/curator/shared/shared_routing.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'collection', type: :routing do |read_only: false|
```

Comment:
Naming these shared examples `collection` may generate some confusion, since we have a `Collection` model. Is there a different title we could give these? `index_create` or something?

Reply:
I was going off the naming convention rails uses for defining routes like 

But if you think that it will still cause confusion i can rename it.
```
resources :foos do
     collection do
         resources :bars
    end
end
```